extension/options/src/App.js

```diff
@@ -0,0 +1,140 @@
+import React, { useEffect, useState } from "react";
+
+const defaultOptions = {
+  arrowParens: "avoid",
+  bracketSpacing: true,
+  jsxBracketSameLine: false,
+  printWidth: 80,
+  semi: true,
+  singleQuote: false,
+  tabWidth: 2,
+  trailingComma: "none",
+  useTabs: false
+};
+
+function App() {
+  const [options, setOptions] = useState();
+
+  useEffect(() => {
+    window.chrome.storage.sync.get(defaultOptions, setOptions);
+  }, [setOptions]);
+
+  useEffect(() => {
+    if (options) {
```

Comment:
When would `options` evaluate to falsy here?

Reply:
`setOptions` is asyncronous, so the `options` state isn't updated until the next render, and remains `undefined` on the first call of this `useEffect` hook.

Reply:
Ah, got it! Thanks for the explanation!